### PR TITLE
Ban 0.13.8.1 from the network

### PR DESCRIPTION
**Pull request by psolstice**

### CODE REVIEW

**Comment by nopslide** [comment]
This is temporary solution right ? Coz someone can rename it from client side.

**Comment by psolstice** [comment]
they can but there is no rational reason to do that

File: src/zerocoin_params.h [hunk 1]
```diff
@@ -42,6 +42,9 @@ static const int64_t DUST_HARD_LIMIT = 1000;   // 0.00001 XZC mininput
 #define ZC_SIGMA_STARTING_BLOCK         182030 //Approx July 23rd, 2019, 8:00 AM UTC
 #define ZC_SIGMA_TESTNET_STARTING_BLOCK 50000
 
+// The block number after which old sigma clients are banned.
+#define ZC_OLD_SIGMA_BAN_BLOCK          181850 //Approx July 21st, 2019, 4:00 AM UTC
```
Comment (levonpetrosyan93) [inline src/zerocoin_params.h]: Need to update this number also I think

Comment (a-bezrukov) [reply src/zerocoin_params.h]: This number is OK - July 21st, early Monday morning.

Comment (thebevrishot) [reply src/zerocoin_params.h]: I think, It should be about 181540

Comment (a-bezrukov) [reply src/zerocoin_params.h]: The comment was wrong. It should be 22nd early morning.

Comment (thebevrishot) [reply src/zerocoin_params.h]: then make sense

**Pull request merged**